driver.go

```diff
@@ -1440,6 +1446,12 @@ func (d *cephRBDVolumeDriver) unmountPath(path string) error {
 	return err
 }
 
+//  ref: https://bugzilla.redhat.com/show_bug.cgi?id=1103792
+func (d *cephRBDVolumeDriver) shutdownXfs(path string) error {
+	_, err := shWithDefaultTimeout("xfs_io", "-x", "-c", "shutdown", path)
```

Comment:
Interesting, I might use this part of your patch.  Will need to read up more on how xfs_io works and also provide a check to ensure it actually is XFS filesystem before attempting.  Thanks